CONTRIBUTING.md

```diff
@@ -142,6 +144,74 @@ Once your reviewer is happy, they will say `/lgtm`, which will apply the
 Your PR will be automatically merged once it has the `lgtm` and `approved`
 labels, does not have any `do-not-merge/*` labels, and all status checks (eg., rebase, tests, DCO) are positive.
 
+### Resolving Conflicts by Rebasing
```

Comment:
We are assuming the contributor is using a fork, aren't we?

If so, I'd make it explicit.